src/server/config.rs

```diff
@@ -19,6 +19,7 @@ const DEFAULT_STATUS_ADDR: &str = "127.0.0.1:20180";
 const DEFAULT_GRPC_CONCURRENCY: usize = 4;
 const DEFAULT_GRPC_CONCURRENT_STREAM: i32 = 1024;
 const DEFAULT_GRPC_RAFT_CONN_NUM: usize = 1;
+const DEFAULT_GRPC_MEMORY_POOL_QUOTA: ReadableSize = ReadableSize(isize::MAX as u64);
```

Comment:
I think this should be a u64, following `DEFAULT_GRPC_STREAM_INITIAL_WINDOW_SIZE`

Reply:
Agree